engine/interpreter/src/main/scala/pl/touk/nussknacker/engine/spel/Typer.scala

```diff
@@ -132,9 +133,9 @@ private[spel] class Typer(classLoader: ClassLoader, commonSupertypeFinder: Commo
       case e: Indexer =>
         current.stack match {
           case TypedClass(clazz, param :: Nil) :: Nil if clazz.isAssignableFrom(classOf[java.util.List[_]]) => valid(param)
-          case TypedClass(clazz, keyParam :: valueParam :: Nil):: Nil if clazz.isAssignableFrom(classOf[java.util.Map[_, _]]) => valid(valueParam)
+          case TypedClass(clazz, keyParam :: valueParam :: Nil) :: Nil if clazz.isAssignableFrom(classOf[java.util.Map[_, _]]) => valid(valueParam)
           case (d: TypedDict) :: Nil => dictTyper.typeDictValue(d, e).map(toResult)
-          case _ => valid(Unknown)
+          case _ => if(disableDynamicPropertyAccess) invalid("Dynamic property access is not allowed") else valid(Unknown)
```

Comment:
We have multiple subclasses of `TypingResult`. I see that under this default there are: 1) `TypedObjectTypingResult` 2) `TypedTaggedValue` 3)`TypedUnion` and 4) `Unknown`. It looks like `TypedTaggedValue` can be handled like it would be without tag